docs/table_design/expression_partitioning.md

```diff
@@ -0,0 +1,256 @@
+# Expression partitioning (recommended)
+
+Since v3.0, StarRocks supports expression partitioning (previously known as automatic partitioning), which is more flexible and user-friendly. This partitioning method is suitable for most scenarios such as querying and managing data based on continuous time ranges or specific categories.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/ab5b8d48-56a8-4301-bf2c-ff89e3aacba6)
v3.0 or v3.1? In v3.0, we have automatic partitioning. In v3.1, we have expression partitioning. Right?